src/Options.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import images from './data.js';
+
+export default class Options extends Component {
+    // generateOptions = () => {
+        
+        // }
+        render() {
+            
+            console.log(this.props.filterOption)
+            return (
+            <div> 
```

Comment:
I'm wondering why you're returning all of the filter options into a non-responsive div that displays on the screen?